src/bundle/Resources/views/content/content_edit/content_create_blocks.html.twig

```diff
@@ -0,0 +1,46 @@
+{% extends 'EzPlatformAdminUiBundle:content/content_edit/base:content.html.twig' %}
+
+{% trans_default_domain 'content_create' %}
+
+{% block details %}
+    <h2 class="text-muted">{{ 'creating_in_language'|trans({'%contentTypeName%': contentType.name, '%languageName%': language.name})|desc('Creating - %contentTypeName% in %languageName%') }}</h2>
+    <h1>
+        <svg class="ez-icon ez-icon-{{ contentType.identifier }}">
+            <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#{{ contentType.identifier }}"></use>
+        </svg>
+        {{ 'new_content_item'|trans({'%contentType%': contentType.name})|desc('New %contentType%') }}
+    </h1>
+
+    <div class="small">
+        {{ contentType.name }} / Parent Location ID: {{ parentLocation.id }}
+    </div>
+    {# @todo remove if statement once getDescription() bug is resolved in kernel #}
```

Comment:
What bug? :)

Reply:
In case no description is provided it will return undefined index error instead od null/exception.